modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -12832,6 +12832,7 @@ no-variations=No Variations
 no-version-was-found=No version was found.
 no-video-preview-available=No video preview available.
 no-views-created=No Views Created
+no-visualization-modes-has-been-defined=No visualization modes has been defined. The Data Set will not be displayed.
```

Comment:
Either use plural `no-visualization-modes-have-been-defined` or singular `no-visualization-mode-has-been-defined`. Then convey that to the value of this key